daemon/policy.go

```diff
@@ -193,7 +193,7 @@ type AddOptions struct {
 	ReplaceWithLabels labels.LabelArray
 }
 
-func (d *Daemon) policyAdd(rules policyAPI.Rules, opts *AddOptions, prefixes []*net.IPNet) (uint64, error) {
+func (d *Daemon) policyAdd(rules policyAPI.Rules, opts *AddOptions, prefixes []*net.IPNet) (uint64, policyAPI.Rules, error) {
```

Comment:
Maybe a comment why the oldRules are needed in this function will help to get context in the future.

Reply:
I think the use of the `oldRules` and the explanation here suffices - https://github.com/cilium/cilium/pull/6998/files#diff-afacd8d4c41a906545f4ec8cde864b7fR295